src/actions/feedback-form.js

```diff
@@ -23,11 +23,11 @@ export function toggleFeedbackValid () {
 /**
  * Submit the user feedback
  * @param {String} theUrl
- * @param {Object} data
+ * @param {Object}
```

Comment:
It looks as though these too params documentation can go, they don't appear to be used no

Reply:
✅  👍